appveyor.yml

```diff
@@ -59,15 +86,10 @@ install:
     - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls true
-    - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
-    - cmd: conda config --add channels conda-forge
+    - cmd: conda install -c http://conda.binstar.org/pelson/channel/development --yes --quiet obvious-ci
+    - cmd: conda config --add channels http://conda.binstar.org/conda-forge
```

Comment:
Please update your conda-smithy and rerender. All mentions of binstar are gone in the latest conda-smithy.